_includes/layout/edition-callout.html

```diff
@@ -24,7 +24,7 @@
 {% endif %}
 {% if page.pwa_only == true %}
```

Comment:
Probably needs to be changed to `page.graphql_only`? This page variable that is set on the whole page. Edition callout on top of the page uses that